src/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { Response } from "node-fetch";
+
+export const checkStatus = async (res: Response) => {
+  if (res.ok) {
+    // res.status >= 200 && res.status < 300
+    return res;
+  } else {
+    console.log(res.status, res.statusText, await res.text());
+    return res;
+  }
+};
+
+/**
+ * Internal function
+ * @ignore
+ */
+export const matchWithWildcard = (input: string, matchWith: string): boolean => {
```

Comment:
More of an improvement: it might be nice to use a package to match a glob pattern (at least for the resource path): `/1/*/0` (and similar)

Reply:
Good idea.  This was lifted straight from the SDK.  I don't know if the API handles globbing in the middle of the string